azure-pipelines-ci.yml

```diff
@@ -34,14 +34,25 @@ jobs:
     workingDirectory: $(Build.ArtifactStagingDirectory)
   - script: $(Build.ArtifactStagingDirectory)\windows-dependencies\vcpkg.exe integrate install
     displayName: 'Install dependencies'
+  - bash: |
+      set -ex
+      cd bin/baseset
+      curl -L https://binaries.openttd.org/extra/opengfx/0.5.2/opengfx-0.5.2-all.zip > opengfx-0.5.2-all.zip
+      unzip opengfx-0.5.2-all.zip
+      rm -f opengfx-0.5.2-all.zip
+    displayName: 'Install OpenGFX'
   - task: VSBuild@1
     displayName: 'Build'
     inputs:
       solution: 'projects/openttd_vs141.sln'
       platform: $(BuildPlatform)
       configuration: Release
       maximumCpuCount: true
-  # Running the regression is currently not possibe via MSVC (console is not redirected)
+  - script: |
+      call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x86
```

Comment:
is x86 also working for 64bit?

Reply:
Yes, it's just to get access to editbin.exe, and the platform doesn't matter here. Would be different if we needed to use compiler or linker.